test/functional/JIT_Test/playlist.xml

```diff
@@ -598,7 +598,7 @@
 		echo $(Q)$(TEST_JDK_BIN)$(D)jitserver doesn't exist; assuming this JDK does not support JITServer and trivially passing the test.$(Q); \
 	fi; \
 	$(TEST_STATUS)</command>
-		<platformRequirements>os.linux,arch.x86,bits.64,vm.cmprssptrs</platformRequirements>
+		<platformRequirements>os.linux,^arch.arm,^arch.aarch64,^arch.s390x,bits.64,vm.cmprssptrs</platformRequirements>
```

Comment:
I am guessing that `^arch.arm` means NOT arm. Is there a a way to include platforms rather than excluding? I would prefer that because if in the future a new platform comes along, it will be automatically included.

Reply:
Yeah `^` means NOT. I don't think there is a way to include multiple platforms instead of excluding. At least I couldn't find any tests that would do that.

Reply:
There are very many examples that use exclusions, so I am ok with the solution proposed here.